internal/semver/semver.go

```diff
@@ -74,10 +74,8 @@ func prereleaseKeys(p string) (alphas []string, numerics []int) {
 }
 
 func prereleaseCmp(a, b string) int {
-	if a == "" {
-		return 1
-	} else if b == "" {
-		return -1
+	if a == "" || b == "" {
```

Comment:
I know you didn't ask me for a review here, but this line is crazy confusing, perhaps a comment?

Reply:
Good call. And thanks for chiming in. Didn't want to suddenly spring a bunch of PRs on you so i assigned to william :)